### ISSUE TITLE
unable to push

### ISSUE THREAD

**Issue by esmith2678** [udacity/course-collaboration-travel-plans#6034]
git push                                                                     remote: Permission to udacity/course-collaboration-travel-plans.git denied to  esmith2678.                                                                    fatal: unable to access 'https://esmith2678@github.com/udacity/course-collabor ation-travel-plans.git/': The requested URL returned error: 403
--

**Comment by celyes** [comment]
It's obvious ... you didn't fork the repo to your GitHub account and of course, you don't have the permission to write to the Udacity account...

so, the solution is:
1. Fork the repo and push the changes there, not here.

if you already forked the repo and it didn't work, then try this:  `git push -f origin master`

**Comment by kotar03** [comment]
@esmith2678 Read Lesson 9 from the core curriculum 2. Developer's Tools if you are taking Full Stack Web Developer Nanodegree Program ;)

**Comment by HolgerSpernau** [comment]
I had the same problem. Would be great, if the description in the Udacity course would be updated.

**Comment by mmishal001** [comment]
I agree @HolgerSpernau, the explanation steps need to be updated to make sure everyone forks and then pushes the changes although it might be obvious. Good learning though because this is going to stick better.

Update: the course is meant to be that way to make us realise this. The next question proves it.
